app/models/explainer.rb

```diff
@@ -106,7 +108,7 @@ def self.search_by_similarity(text, language, team_id)
         language: language
       }
     }
-    response = Bot::Alegre.request('post', '/text/similarity/search/', params)
+    response = Bot::Alegre.index_async_with_params(params, "text")
```

Comment:
This should be `get_async_with_params`, no?

Reply:
thats right - will fix